public/app/features/explore/GraphContainer.tsx

```diff
@@ -54,7 +58,7 @@ export class GraphContainer extends PureComponent<GraphContainerProps> {
             height={graphHeight}
             id={`explore-graph-${exploreId}`}
             onChangeTime={this.onChangeTime}
-            range={timeRange}
+            range={absolutRange}
```

Comment:
```suggestion
            range={absoluteRange}
```

Reply:
I seem to have some issues with that word, yeah